src/color.rs

```diff
@@ -262,6 +258,48 @@ pub trait ColorComponentParser<'i> {
             ref t => return Err(location.new_unexpected_token_error(t.clone())),
         })
     }
+
+    /// Parse a `<hue>` value.
+    /// https://w3c.github.io/csswg-drafts/css-color-4/#typedef-hue
+    fn parse_hue<'t>(
```

Comment:
This function doesn't really parse hue values, but angles.  Should there be a specific abstraction for hue values only?